packages/design-system/src/components/SearchBar/style.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../scss/variables/colors';
+
+$base-class: "search-bar";
+
+// TODO: should container have fixed width and height?
+.#{$base-class}__container {
+  display: inline-flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  height: 32px;
+  width: 290px;
+}
+
+.#{$base-class}__icon {
+  position: absolute;
+  align-self: center;
+
+  &--search {
+    @extend .#{$base-class}__icon;
+    left: 0;
+    padding: 7px 13px;
+
+    &-compact {
+      cursor: pointer;
+    }
+  }
+
+  &--close {
+    @extend .#{$base-class}__icon;
+    right: 13px;
+    cursor: pointer;
+  }
```

Comment:
It should be `--clear`

Reply:
Also, accessibility states (especially focus) should be set for this.

Reply:
I've changed it to `--clear`. I didn't add any accessiblity states, because frankly i don't know how they should look like. Should I discuss it with designer? On focus I see that there is blue outline near clear button "X"

Reply:
Use the same style as in the Banner component